packages/atomic/tailwind.config.js

```diff
@@ -50,9 +50,9 @@ module.exports = {
         1: '1',
       },
       animation: {
-        scaleUpRefineModal:
+        scaleUpModal:
           'scaleUp .5s cubic-bezier(0.165, 0.840, 0.440, 1.000) forwards',
-        slideDownRefineModal: 'slideDown .5s linear forwards',
```

Comment:
I don't think developers were able to overwrite these, so I see no harm in renaming them.